internal/datastore/elastic/elastic.go

```
@@ -46,25 +71,104 @@ func (e *ElasticRepository) Write(ctx context.Context, event entity.Event) error
 	return nil
 }
 
-func (e *ElasticRepository) CreateIndex(name string) error {
-	// check if index exists
-	res, err := e.client.Indices.Exists([]string{name})
+func (e *ElasticRepository) updateByQuery(ctx context.Context, req UpdateByQueryRequest) (*UpdateByQueryResult, error) {
```

golangci-lint (2.12.2)

[error] func (*ElasticRepository).updateByQuery is unused

(unused)